Modules/_zstd/compressor.c

```diff
@@ -198,7 +198,7 @@ _get_CDict(ZstdDict *self, int compressionLevel)
 }
 
 int
```

Comment:
```suggestion
static int
```

See above about static linkage.